app.js

```diff
@@ -85,6 +201,64 @@ app.action('intro_progress', async ({ ack, body }) => {
   })
 });
 
+
+app.action('they', async ({ ack, body }) => {
+  ack();
+  pronouns = "They/them/theirs"
+  pronoun_1 = "They"
+  updateInteractiveMessage(body.message.ts, body.channel.id, `Awesome! Taken note of. Every profile here has a custom field for pronouns, I recommend you add yours there to let others know! <${`https://slack.com/intl/en-sg/help/articles/204092246-Edit-your-profile`}|Here is a quick tutorial.>`)
+  console.log(pronouns)
+  await timeout(3000)
+  await app.client.chat.postMessage({
+    token: process.env.SLACK_BOT_TOKEN,
+    channel: body.channel.id,
+    blocks: [
+      {
+        "type": "section",
+        "text": {
+          "type": "mrkdwn",
+          "text": `Are you currently a high school student? (it's OK if you're not)`
+        }
+      },
+      {
+        "type": "actions",
+        "elements": [
+          {
+            "type": "button",
+            "text": {
+              "type": "plain_text",
+              "emoji": true,
+              "text": "Yes"
+            },
+            "style": "primary",
+            "action_id": "hs_yes"
+          },
+          {
+            "type": "button",
+            "text": {
+              "type": "plain_text",
+              "emoji": true,
+              "text": "No"
+            },
+            "style": "danger",
+            "action_id": "hs_no"
+          }
+        ]
+      }
+    ]
+  })
+});
+
+app.action('something_else', async ({ ack, body }) => {
+  ack();
+  
+  await sendMessage(body.channel.id, `Awesome! Could you please elaborate on how would you like us to address you in the format of They/Them/Theirs?`)
+  
+});
+
+app.action('hs-question', async ({ ack, body }) => {
```

Comment:
This is the only hyphenated action name & it doesn’t do anything, just want to check that those are intentional?

Reply:
oh oops i left that in